### PR TITLE
Add visual feedback for different transaction list states

### LINKED ISSUE #359: Add spinner while rescanning
Following rescan, It takes approx. 30 seconds for a transaction list to appear.  May need to add a spinner/wait icon.

### PR DESCRIPTION
**Pull request by pablen**
Transaction list up-to-date:
captura de pantalla 2018-07-23 a la s 18 04 24

Transaction list syncing:
captura de pantalla 2018-07-23 a la s 18 06 37

Empty transaction list syncing:
captura de pantalla 2018-07-23 a la s 18 07 45

Wallet offline:
captura de pantalla 2018-07-23 a la s 18 05 00

Closes #359

### CODE REVIEW

File: public/main/plugins/ethWallet/index.js [hunk 1]
```diff
@@ -363,6 +363,7 @@ function syncTransactions ({ number, walletId, webContents, bloqEthExplorer }) {
       logger.debug('Syncing', addresses, indexed)
       return Promise.all(
         addresses.map(function (address) {
+          webContents.send('transactions-scan-started', { address })
```
Comment (pablen) [inline public/main/plugins/ethWallet/index.js:366]: @gabmontes check the changes in this file!
Seemed simple enough to add the required messages, but maybe I overlooked something...

Comment (gabmontes) [reply public/main/plugins/ethWallet/index.js:366]: Shouldn't we consider the case the `'transactions-scan-started'` is sent, then the promise chain breaks and the `'transactions-scan-finished'` is never sent? Perhaps adding that event to the final `catch` down below.

**Review by ndelvalle — DISMISSED** [review]

File: public/main/plugins/ethWallet/index.js (hunk 1 above)
Comment (pablen) [reply public/main/plugins/ethWallet/index.js:366]: good `catch`! (pun intended)

**Comment by pablen** [comment]
Added commit to address @gabmontes comment.

**Review by ianaya89 — APPROVED** [review]

**Review by gabmontes — APPROVED** [review]

**Pull request merged**